src/harness/unittests/programMissingFiles.ts

```diff
@@ -3,7 +3,7 @@
 namespace ts {
     function verifyMissingFilePaths(missingPaths: ReadonlyArray<Path>, expected: ReadonlyArray<string>) {
         assert.isDefined(missingPaths);
-        const map = arrayToMap(expected, k => k, _v => true);
+        const map = arrayToSet(expected) as Map<boolean>;

```

Comment:
Would keep this as a set, and `map.delete(missing)` instead of `map.set(missing, false)`. Then `if (map.size) throw new Error(`Not found ${arrayFrom(map.keys())} in actual: ${missingPaths} expected: ${expected}`);`